app/vmalert/static/js/custom.js

```diff
@@ -0,0 +1,40 @@
+function expandAll() {
+    $('.collapse').addClass('show');
+}
+
+function collapseAll() {
+    $('.collapse').removeClass('show');
+}
+
+function toggleByID(id) {
+    debugger;
+    let el = $("#" + id);
+    if (el.length > 0) {
+        el.click();
+    }
+}
+
+$(document).ready(function () {
+    $(".group-heading a").click(function (e) {
+        e.stopPropagation(); // prevent collapse logic on link click
+        let target = $(this).attr('href');
+        if (target.length > 0) {
+            toggleByID(target.substr(1));
+        }
+    });
+
+    $(".group-heading").click(function (e) {
+        let target = $(this).attr('data-bs-target');
+        let el = $("#" + target);
+        new bootstrap.Collapse(el, {
+            toggle: true
+        });
+    });
+
+    let hash = window.location.hash.substr(1);
+    toggleByID(hash);
+});
+
```

Comment:
You can remove this part of the code and add specific data attrs to the root element. In your case, it would be an element with the class `group-heading`.
Some Example
```javascript
<div class="group-heading" data-bs-toggle="collapse" data-bs-target="#notifiers-{%s typeK %}">
```
And all collapse logic should work as expected

Reply:
Will check and maybe find an improvement solution :)